gslib/utils/iam_helper.py

```diff
@@ -95,18 +95,22 @@ def DeserializeBindingsTuple(serialized_bindings_tuple):
 
 
 def BindingsToDict(bindings):
-  """Converts a list of BindingsValueListEntry to a dictionary.
+  """Reformats policy bindings metadata.
 
   Args:
-    bindings: A list of BindingsValueListEntry instances.
+    bindings: Either a list of BindingsValueListEntry instances,
+    or a list of analogous dictionaries.
```

Comment:
Can you add the expected structure of the dictionaries? Mentioned in another comment that maybe there should be separate functions for dicts and Apitools objects like `BindingsValueListEntry`

Reply:
Done.